devito/ir/support/basic.py

```diff
@@ -364,11 +365,13 @@ def distance(self, other):
                 # trip count. E.g. it ranges from 0 to 3; `other` performs a
                 # constant access at 4
                 for v in (self[n], other[n]):
-                    try:
-                        if bool(v < sit.symbolic_min or v > sit.symbolic_max):
-                            return Vector(S.ImaginaryUnit)
-                    except TypeError:
-                        pass
+                    # Note: To avoid evaluating expensive symbolic Lt or Gt operations,
```

Comment:
this comment is now obsolete and should be dropped

smart_lt's docstring should already contain all this info, no?